build-scripts/bundle.js

```diff
@@ -2,6 +2,11 @@ const path = require("path");
 const env = require("./env.js");
 const paths = require("./paths.js");
 
+// GitHub base URL to use for production source maps
+// Assumes there is a tag that matches the version
+module.exports.sourceMapURL = () =>
+  `https://raw.githubusercontent.com/home-assistant/frontend/${env.version()}`;
```

Comment:
This will not work for nightly builds, as we don't tag them, should we tag when we do a nightly build, or change this to use the `dev` branch when the version includes `dev`?

Reply:
No need.  I'll just change it to use the commit SHA.